mica-rest/src/main/java/org/obiba/mica/AbstractGitPersistableResource.java

```diff
@@ -65,6 +65,9 @@
   @Inject
   private CommentResource commentResource;
 
+  @Value("${portal.url}")
```

Comment:
I guess that it comes from the application properties. Would prefer to have it from the MicaConfig (=with administration UI).